modules/invoicer_invoice/config/install/invoicer_invoice.settings.yml

```diff
@@ -0,0 +1,40 @@
+vat_types:
```

Comment:
@mmanzano can you change this file to these format, please:
´´´
vat_types:
  - 0:
    value: 0
    value_label: '0%'
  - 1:
    value: 3
    value_label: '3%'
  - 4:
    value: 4
    value_label: '4%'
  - 6:
    value: 6
    value_label: '6%'
  - 7:
    value: 7
    value_label: '7%'
  - 8:
    value: 8
    value_label: '8%'
  - 10:
    value: 10
    value_label: '10%'
  - 12:
    value: 12
    value_label: '12%'
  - 13:
    value: 13
    value_label: '13%'
  - 15:
    value: 15
    value_label: '15%'
  - 16:
    value: 16
    value_label: '16%'
  - 18:
    value: 18
    value_label: '18%'
  - 21:
    value: 21
    value_label: '21%'
´´´